hotsos/cli.py

```diff
@@ -32,6 +33,11 @@ def get_repo_info():
         with open(repo_info) as fd:
             return fd.read().strip()
 
+    # pypi
+    with resources.path('hotsos', '.repo-info') as repo_info:
+        with open(repo_info) as fd:
+            return fd.read().strip()
```

Comment:
When we run the `functional` test the `.repo-info` file does not exist.

```suggestion
    try:
        with resources.path('hotsos', '.repo-info') as repo_info:
            with open(repo_info) as fd:
                return fd.read().strip()
    except FileNotFoundError:
        return "unknown"
```